src/distributions/temurin/installer.ts

```diff
@@ -152,4 +153,15 @@ export class TemurinDistribution extends JavaBase {
         return process.platform;
     }
   }
+
+  private osArchToDistributionArch(osArch: string): string {
+    let dArch;
+    switch (osArch) {
+      case 'amd64': dArch = 'x64'; break;
+      case 'ia32': dArch = 'x32'; break;
+      case 'arm64': dArch = 'aarch64'; break;
+      default: dArch = osArch;
+    }
+    return dArch;
+  }
```

Comment:
I think dArch variable is not needed

Reply:
You mean just return from the cases instead? I can do that.

Reply:
```
 private osArchToDistributionArch(osArch: string): string {
    switch (osArch) {
      case 'amd64': osArch = 'x64'; break;
      case 'ia32': osArch = 'x32'; break;
      case 'arm64': osArch = 'aarch64'; `break;`
    }
    return osArch;
  }
```

Just to be consistent with other actions. And some of checks are failing, can you rebase to main, build and push again.

Reply:
There weren't any new commits to main to rebase onto.